smf/sled-agent/gimlet-standalone/config.toml

```diff
@@ -14,6 +14,14 @@ sled_mode = "scrimlet"
 # this information.
 sidecar_revision = "b"
 
+# Assigns a unique name to the sidecar, which is used to tag its data in
+# oximeter.
+# TODO: we need a way to generate a UUID for a sidecar that is actually unique
+# and persists across reboots.  Presumably we want this UUID to be unique to
+# this switch and rack combination, so a new UUID will to be generated when the
+# switch is replaced or moved to a new rack.
+sidecar_name = "8f280386-1e33-482e-bf30-948a4e36790d"
```

Comment:
These are all called `sidecar_name`, but isn't the value called `sidecar_uuid`? Are these getting parsed correctly?

Reply:
They are not at the moment.  I made a late change in one place but not the other.  Thanks.